auth/service.bal

```diff
@@ -0,0 +1,101 @@
+import ballerina/http;
+import ballerina/crypto;
+import ballerina/uuid;
+import e_backend.data;
+import ballerina/persist;
+import ballerina/sql;
+
+listener http:Listener voterListener = new (9090);
+
+final data:Client dbClient = check new ();
+
+
+public type VoterLogin record { 
+    string nationalId; 
+    string password; 
+};
+
+service /Voter\-registration/api/v1 on voterListener {
+
+    // Register a new voter
+    @http:ResourceConfig {}
+    resource function post voters/register(@http:Payload data:Voter newVoter)
+            returns http:Created|http:Forbidden|error {
+
+        string hashedPassword = crypto:hashMd5(newVoter.password.toBytes()).toBase16();
```

Comment:
Pass a salt value as well. To learn about salting refer - https://www.paubox.com/blog/what-is-password-salting